trunk/SUAVE/Methods/Weights/Correlations/Common/weight_transport.py

```diff
@@ -0,0 +1,403 @@
+            raise ValueError("Lenght of wing_mounted list does not match number of engines")
+
+    # Set the factors
+    if settings is None:
+        wt_factors = Data()
+        wt_factors.main_wing    = 0.
+        wt_factors.empennage    = 0.
+        wt_factors.fuselage     = 0.
+        wt_factors.structural   = 0.
+        wt_factors.systems      = 0.
+    else:
+        wt_factors = settings.weight_reduction_factors
+        if 'structural' in wt_factors and wt_factors.structural != 0.:
+            print('Overriding individual structural weight factors')
+            wt_factors.main_wing    = 0.
+            wt_factors.empennage    = 0.
+            wt_factors.fuselage     = 0.
+            wt_factors.systems      = 0.
+        else:
+            wt_factors.structural   = 0.
+            wt_factors.systems      = 0.
+
+    # Prop weight
+    wt_prop_total   = 0
+    wt_prop_data    = None
+    for prop in vehicle.propulsors:
+        if isinstance(prop, Nets.Turbofan) or isinstance(prop, Nets.Turbojet_Super) or isinstance(prop,
+                                                                                                  Nets.Propulsor_Surrogate):
+            num_eng     = prop.number_of_engines
+            thrust_sls  = prop.sealevel_static_thrust
+            if 'total_weight' in prop.keys():
+                wt_prop         = prop.total_weight
+            elif method_type == 'FLOPS Simple' or method_type == 'FLOPS Complex':
+                wt_prop_data    = total_prop_flops(vehicle, prop)
+                wt_prop         = wt_prop_data.wt_prop
+            elif method_type == 'Raymer':
+                wt_prop_data    = total_prop_Raymer(vehicle, prop)
+                wt_prop         = wt_prop_data.wt_prop
+
+            else:
+                wt_engine_jet = Propulsion.engine_jet(thrust_sls)
+                wt_prop       = Propulsion.integrated_propulsion(wt_engine_jet, num_eng)
+
+            if num_eng == 0.:
+                wt_prop = 0.
+            prop.mass_properties.mass = wt_prop
+            wt_prop_total += wt_prop
+
+    # Payload Weight
+    if method_type == 'FLOPS Simple' or method_type == 'FLOPS Complex':
+        payload = payload_FLOPS(vehicle)
+    else:
+        payload = payload_weight(vehicle)
+    vehicle.payload = payload.total
+    # Operating Items Weight
+    if method_type == 'FLOPS Simple' or method_type == 'FLOPS Complex':
+        wt_oper = operating_system_FLOPS(vehicle)
+    else:
+        wt_oper = operating_system(vehicle)
+
+    # System Weight
+    if method_type == 'FLOPS Simple' or method_type == 'FLOPS Complex':
+        wt_sys = systems_FLOPS(vehicle)
+    elif method_type == 'Raymer':
+        wt_sys = systems_Raymer(vehicle)
+    else:
+        wt_sys = systems(vehicle)
+    for item in wt_sys.keys():
+        wt_sys[item] *= (1. - wt_factors.systems)
+
+    WPOD = 0.0
+    if method_type == 'FLOPS Complex':
```

Comment:
Can you explain what you're looking to do here?

Reply:
I wanted the pod weights along with the propulsor weights. This script is rather long, so I think it's a more logical way to group things for later debugging. The reason this is done after the propulsor weights, is because the pod uses system weights.

Reply:
I tried and it appears the propulsion calculations must be done before systems. I added a comment to make this easier to find.